playwright/tests/organize/people/views/columns/create-column.spec.ts

```diff
@@ -77,7 +77,7 @@ test.describe('Creating a view column', () => {
         });
 
         // Create new toggle column
-        await page.click('data-testid=create-column-button');
+        await page.click('data-testid=ViewDataTableToolbar-createColumn');
```

Comment:
I like this new pattern

Reply:
I'm glad! I was struggling a bit to come up with names that felt namespaces to the context they're in (there could potentially be two buttons on a page to create a column) but still not too long. And then I realized this could be a way to do it.